fdbserver/DataDistribution.actor.cpp

```diff
@@ -2766,6 +2770,30 @@ ACTOR Future<Void> serverTeamRemover(DDTeamCollection* self) {
 	}
 }
 
+ACTOR Future<Void> zeroServerLeftLogger_impl(DDTeamCollection* self, Reference<TCTeamInfo> team) {
+	wait(delay(SERVER_KNOBS->DD_TEAM_ZERO_SERVER_LEFT_LOG_DELAY));
+	state vector<KeyRange> shards = self->shardsAffectedByTeamFailure->getShardsFor(
+	    ShardsAffectedByTeamFailure::Team(team->getServerIDs(), self->primary));
+	state std::vector<Future<StorageMetrics>> sizes;
+	sizes.reserve(shards.size());
+
+	for (auto const& shard : shards) {
+		sizes.emplace_back(brokenPromiseToNever(self->getShardMetrics.getReply(GetMetricsRequest(shard))));
+		TraceEvent(SevError, "DDShardLost").detail("ShardBegin", shard.begin).detail("ShardEnd", shard.end);
```

Comment:
TraceEvent(SevError, "DDShardLost") -->
TraceEvent(SevWarnAlways, "DDShardLost", self->distributorId)

Reply:
Can we generate a UID and associate it with  team->getDesc()? 
In each "DDShardLost" msg, we also track the UID. 

This is helpful to know which shards are on which team when multiple server teams are lost.